examples/3d/lighting.rs

```diff
@@ -1,3 +1,5 @@
+//! Illustrates various lighting options in a simple scene
```

Comment:
I think the description should talk about the fact that there are used both different light types (point, directional, ...) and different colors, instead of simply stating “various lighting options”.

Reply:
Yeah, sounds good. I'll try to come up with something more helpful!